src/types.ts

```diff
@@ -103,7 +103,7 @@ type UserAction =
 type BackgroundAction =
   | { type: 'fetchHandleStart'; payload: { tabId: number } }
   | { type: 'fetchHandleSuccess'; payload: { tabId: number; host: string; handle: string } }
-  | { type: 'fetchHandleFailure'; payload: { tabId: number; host: string; error: any } }
+  | { type: 'fetchHandleFailure'; payload: { tabId: number; host: string; alert: string } }
```

Comment:
We'd want to do this for all actions ending in `Failure` which include an `error`